$NetBSD: patch-aa,v 1.1.1.1 2009/08/25 22:18:03 leot1990 Exp $

test(1) does not support "==", use "=" instead to compare if two strings are
identical.

--- ppd/compile.sh.orig	2009-02-06 12:49:09.000000000 +0100
+++ ppd/compile.sh
@@ -28,7 +28,7 @@ parseFile() {
 #
 # Main script
 #
-if [ $2 == "debug" ]; then
+if [ $2 = "debug" ]; then
     TMPFILE="output.drv"
     DRIVER=$1
     shift 1
@@ -37,7 +37,7 @@ if [ $2 == "debug" ]; then
     parseFile $DRIVER $TMPFILE
 
 
-elif [ $2 == "lang" ]; then
+elif [ $2 = "lang" ]; then
     if [ -z $TMP ]; then
         TMP="/tmp"
     fi;
